WebRoot/public/modern.jsp

```diff
@@ -8,6 +8,20 @@
 %>
 <script TYPE="text/javascript">
     localStorage.setItem("csrfToken" , "${csrfToken}");
-    var url = window.location.origin + "/modern/"
-    window.location.href = url;
+    
+    const currentURL = new URL(window.location.href);
+    const redirectToPath = currentURL.searchParams.get("RelayState") || '';
```

Comment:
Lots of mixing of `'` and `"` in here. Does this pass linter?

Reply:
unfortunately we don't have linters in classic ui code